controlador/jogador_ctrl.py

```diff
@@ -39,19 +39,58 @@ def mostrar_jogador_logado(self):
         if jogador:
             self.__jogador_tela.mostra_perfil_jogador(jogador)
             opcoes_acoes = {
-                1: self.editar_jogador,
-                2: self.excluir_jogador,
-                3: self.__controlador_principal.iniciar_app,
+                1: self.historico_jogos,
+                2: self.editar_jogador,
+                3: self.excluir_jogador,
+                4: self.__controlador_principal.iniciar_app,
             }
 
             opcao_escolhida = self.__jogador_tela.mostra_menu_perfil()
             opcoes_acoes[opcao_escolhida]()
 
+    def tratar_usario(self) -> str:
+        usuarios = [jogador.usuario for jogador in self.jogadores]
+        while True:
+            usuario = self.__jogador_tela.obtem_informacao(
+            'Digite seu usuário: ').strip().lower()
+            if usuario not in usuarios:
+                return usuario
+            else:
+                self.__jogador_tela.mostra_mensagem(
+                    'Nome de usuário ja está em uso...')
+
+    def tratar_data_nascimento(self) -> str:
+        while True:
+            try:
+                dia, mes, ano = self.__jogador_tela.obtem_informacao(
+                    f'Digite sua data de nascimento '
+                    f'separada por espaços: ').split()
+                dia = int(dia)
+                mes = int(mes)
+                ano = int(ano)
+                if not (0 < dia < 32):
+                    raise ValueError
+                elif not (0 < mes < 13):
+                    raise ValueError
+                elif not (ano > 0):
```

Comment:
ao invés de usar vários if's da pra colocar tudo em um só, já que dão raise no mesmo tipo de erro, dai as condicionais vão ser todas unificadas, ex: `not (0 < dia < 32) or not (0 < mes < 13 ) or (ano < 0)`

se precisar pra ficar mais bonitinho só quebrar a linha nas condicionais